docs/source/index.rst

```diff
@@ -5,67 +5,62 @@
 
 .. role:: red
 
-Welcome to the Docs for Forest and pyQuil!
-==========================================
+Welcome to the Docs for the Forest SDK!
+=======================================
 
+The Rigetti Forest `Software Development Kit <http://rigetti.com/forest>`_ includes pyQuil, our Quil Compiler (quilc),
+and our QVM.
 
-PLEASE READ: A NOTE FROM RIGETTI COMPUTING
-==========================================
-.. note::
-      Our forthcoming flagship product, `Quantum Cloud Services <http://rigetti.com/qcs>`_, is an overhaul of pyQuil,
-      Quil, compilation, access, and execution. Due to the significance of the changes, programs written in pyQuil v1.9
-      will need refactoring before they will run on the new QVM, Quil Compiler or QPU. Our 1.3-based QPUs are now
-      offline; the QPUs that replace them will only be compatible with pyQuil v2.0 and beyond. For now, you will still
-      be able to run programs against our cloud-based QVM, but we encourage that you transition to our SDK, which
-      includes a downloadable QVM and QUILC (thus eliminating the need for you to send jobs to a cloud queue just to run
-      programs). To download our new SDK Preview, see the :ref:`sdk`. Current users: please refer to the :ref:`quickstart`
-      after downloading to get caught up on changes.
+**Longtime users of Rigetti Forest will notice a few changes.** First, the SDK now contains a downloadable compiler and a
+QVM. Second, the SDK contains pyQuil 2.0, with significant updates to previous versions. As a result, programs written
+using previous versions of the Forest toolkit will need to be updated to pyQuil 2.0 to be compatible with the QVM or compiler.
 
+Scroll down to the :ref:`quickstart` to get caught up on what's new!
 
-Overview
---------
+Quantum Cloud Services will provide users with a dedicated Quantum Machine Image, which will come prepackaged with the
+Forest SDK. We’re releasing a Preview to the Forest SDK now, so current users can begin migrating code (and share feedback
+with us early and often!). Longtime Forest users should start with the Migration Guide which outlines key changes in this
+SDK Preview release.
+
+If you’re new to Forest, we hope this documentation will provide everything you need to get up and running with the toolkit.
+Once you’ve oriented yourself here, proceed to the section :ref:`start` to get started. Once you’ve installed the SDK,
+take a look at our section on :ref:`intro`. There, you’ll learn the basic concepts needed to write quantum software. You
```

Comment:
this paragraph seems to imply that we suggest going to `intro` after `start`; however, `intro` is for some reason at the very bottom of the toctree rather than immediately following `start`

Reply:
Good point, rephrased.